app/experimenter/static/rapid/components/experiments/ExperimentDetails.tsx

```diff
@@ -131,34 +131,35 @@ const ExperimentDetails: React.FC = () => {
     );
   }
 
-  const buttonsDisabled = experimentData.status !== ExperimentStatus.DRAFT;
-  let buttonsClass = "btn btn-primary";
-  if (buttonsDisabled) {
-    buttonsClass = "btn btn-secondary";
-  }
+  const approveButtonDisabled =
+    experimentData.status !== ExperimentStatus.DRAFT;
+  const backButtonDisabled = ![
+    ExperimentStatus.DRAFT,
+    ExperimentStatus.REJECTED,
+  ].includes(experimentData.status);
 
   const buttonsShown = ![
     ExperimentStatus.LIVE,
     ExperimentStatus.COMPLETE,
   ].includes(experimentData.status);
 
   let changeStatusButtons;
+
   if (buttonsShown) {
     changeStatusButtons = (
       <div className="d-flex mt-4">
         <span>
-          <Link
-            className={buttonsClass}
-            to={buttonsDisabled ? "#" : `/${experimentData.slug}/edit/`}
-          >
-            Back
+          <Link to={`/${experimentData.slug}/edit/`}>
+            <button className="btn btn-primary" disabled={backButtonDisabled}>
```

Comment:
Why did you change this to a button?

Reply:
Oh I didn't notice she did that, that's clever!  This is so it gets properly disabled rather than 'fake' disabled which was driving @pdehaan nuts

Reply:
Oh it doesn't actually solve the problem, it still shows the hand cursor.

Reply:
Sigh we probably have to write some css for that.

Reply:
i'll take a look when i run it

Reply:
Ok, this is what I'd do:

```ts
<Link
  className={
    "btn btn-primary" + (backButtonDisabled ? " disabled" : "")
  }
  to={`/${experimentData.slug}/edit/`}
  onClick={(e) => backButtonDisabled && e.preventDefault()}
  >
  Back
</Link>
```